backend/src/database/user-content/user-content.service.ts

```diff
@@ -105,6 +105,27 @@ export class UserContentService {
     };
   }
 
+  /**
```

Comment:
This implementation is doubled with the implementation in #228 , this will make merge conflics, but this implementation is better.